core/src/apps/cardano/layout.py

```diff
@@ -371,44 +360,29 @@ async def confirm_stake_pool_metadata(
     )
 
 
-async def confirm_transaction_network_ttl(
+async def confirm_stake_pool_registration_final(
     ctx: wire.Context,
     protocol_magic: int,
     ttl: int | None,
     validity_interval_start: int | None,
 ) -> None:
     await confirm_properties(
         ctx,
-        "confirm_pool_network",
+        "confirm_pool_final",
         title="Confirm transaction",
         props=[
-            ("Network:", protocol_magics.to_ui_string(protocol_magic)),
-            (
-                "Valid since: %s" % format_optional_int(validity_interval_start),
-                None,
-            ),
+            ("Confirm signing the stake pool registration as an owner\n\n\n", None),
```

Comment:
I removed the newlines and I also changed the other properties to use the tuple instead of string formatting - it seems more correct in the code and doesn't mess up the UI too much. https://github.com/trezor/trezor-firmware/commit/45b53e8adebc8240d508e07df799b5e1a1de3ee8

The newlines are used similarly multiple times in the file though so that's where I got it from --> https://github.com/trezor/trezor-firmware/blob/master/core/src/apps/cardano/layout.py#L153. This was added when transitioning to the new layout system.

Reply:
Yeah, those shouldn't be there either, they're from a previous iteration of the layout system. We'll be cleaning up all of it within this year.

:+1: for tuple usage. The reason we left the original formatting was to keep the UI differences to a minimum, but this is another thing we'll be flipping over soon.